1.1/spec/20-Annex-E.adoc

```diff
@@ -237,7 +237,8 @@ DCTERMS Source: https://www.dublincore.org/specifications/dublin-core/dcmi-terms
 | <<Property: geo:hasGeometry, `geo:hasGeometry`>> | http://www.w3.org/2000/01/rdf-schema#subPropertyOf[`rdfs:subPropertyOf`] | http://purl.org/dc/terms/spatial[`dcterms:spatial`] | http://purl.org/dc/terms/spatial[`dcterms:spatial`] indicates the "Spatial characteristics of the resource", thus it is a more general form of GeoSPARQL's <<Property: geo:hasGeometry, `geo:hasGeometry`>> which indicates geometry spatial information
 |===
 
-* http://purl.org/dc/terms/coverage[`dcterms:coverage`] is extremely generic - "The spatial or temporal topic of the resource, spatial applicability of the resource, or jurisdiction under which the resource is relevant." - but DCTERMS indicates its range includes a http://purl.org/dc/terms/Location[`dcterms:Location`], so it is a property for indicating a <<Class: geo:Feature, `geo:Feature`>>, not a <<Class: geo:Geometry, `geo:Geometry`>> and for which GeoSPARQL has no equivalent. Often, http://purl.org/dc/terms/coverage[`dcterms:coverage`] is used to indicate a spatial extent such as a bounding box but http://purl.org/dc/terms/spatial[`dcterms:spatial`] could be used for this with more precsision. GeoSPARQL now provides a <<Property: geo:hasBoundingBox, `geo:hasBoundingBox`>> property, so such a property could be used if a Bounding Box is wanted to be indicated.
+* http://purl.org/dc/terms/spatial[`dcterms:spatial`]: "Spatial characteristics of the resource". The range of this property includes a http://purl.org/dc/terms/Location[`dcterms:Location`], so it is a property for indicating a <<Class: geo:Feature, `geo:Feature`>>, for which GeoSPARQL has no equivalent, but perhaps also for indicating a <<Class: geo:Geometry, `geo:Geometry`>>, thus the subProertyOf mapping above.
```

Comment:
typo 'subProertyOf'

Reply:
Thanks, fixed in https://github.com/opengeospatial/ogc-geosparql/pull/363/commits/289bda4e58da958f8487b926117ebd3bb1ced467